FEM/invariants.h

```diff
@@ -0,0 +1,131 @@
+#ifndef INVARIANTS_H
+#define INVARIANTS_H
+
+#include <cfloat>
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+
+#include "Eigen/Dense"
+
+namespace SolidMath
+{
+
+static Eigen::Matrix<double,6,6> P_dev; //deviatoric projection matrix
+static Eigen::Matrix<double,6,6> P_sph; //spherical projection matrix
+static Eigen::Matrix<double,6,6> ident; //identity matrix
+static Eigen::Matrix<double,6,1> ivec; //Kelvin mapping of 2nd order identity
+
+static void InitialiseProjectionTensors()
```

Comment:
remove `static` keywords in all the functions here. The keyword means the functions should be used only in this file, which is not your intention I think. 

and please move implementations of the functions to cpp file.